### ISSUE TITLE
Have f-test model 

### ISSUE THREAD

**Issue by amichuda** [s3alfisc/wildboottest#26]
```       
     return np.nan
```

https://github.com/amichuda/statsmodels/blob/c74f4a6e3b593ad4c2ec4dc4adc1a36d8ab48cf4/statsmodels/regression/linear_model.py#L1853

Right now we don't fun a f-stat for the full model, but the statsmodels table has it. Setting it to np.nan for now.

**Comment by s3alfisc** [comment]
Yes, sounds good. We can have it once we have access to the full vcov matrix.
